docs/DEVELOPER.md

```diff
@@ -48,3 +50,7 @@ control the exposed data model.
 current value from a live instance (used by `export` feature). Here again JCasC offers as default
 implementation a JavaBean compliant implementation, but one could override get and/or set operation with custom
 code to support alternate mechanisms.
+
+## Useful links
+
+* [Javadoc](https://javadoc.jenkins.io/plugin/configuration-as-code/)
```

Comment:
is the tests javadoc uploaded?

Reply:
nope. We can add it to https://github.com/jenkins-infra/javadoc/blob/master/scripts/generate-javadoc-components.groovy tho

Reply:
FTR it would require code update to support custom classifiers